src/main/java/stream/flarebot/flarebot/util/pagination/PagedEmbedBuilder.java

```diff
@@ -12,6 +13,8 @@
     private PaginationList<T> list;
     private boolean hasCodeBlock = false;
     private Color color;
+    private String groupPrefix = null;
+    private int groupsPerPage = 1;
```

Comment:
1 good for a default?

Reply:
I imagine in the most case people are going to have more then one thing in a group and will want it split out so groups are on diffrent pages